tests/system/test_system.py

```diff
@@ -67,7 +68,9 @@ def _list_entries(logger):
     :returns: List of all entries consumed.
     """
     inner = RetryResult(_has_entries, max_tries=9)(_consume_entries)
-    outer = RetryErrors((ServiceUnavailable, ResourceExhausted), max_tries=9)(inner)
+    outer = RetryErrors(
```

Comment:
I think this is OK for the purposes of tests. For any changes to automatic retries in the client make sure to reference https://google.aip.dev/194 (it has guidance on error codes that should/shouldn't be retried).

Reply:
Good to know, thanks!